homeassistant/components/motionmount/select.py

```diff
@@ -0,0 +1,60 @@
+"""Support for MotionMount numeric control."""
+import motionmount
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, WALL_PRESET_NAME
+from .entity import MotionMountEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Vogel's MotionMount from a config entry."""
+    mm = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities((MotionMountPresets(mm, entry),), True)
+
+
+class MotionMountPresets(MotionMountEntity, SelectEntity):
+    """The presets of a MotionMount."""
+
+    _attr_translation_key = "motionmount_preset"
+    _attr_current_option: str | None = None
```

Comment:
```suggestion
    _attr_current_option = None
```

Reply:
This gives an error in the pre-commit hook on line 51 and 57 assigning `str` to `None`.